libraries/c_sdk/standard/mqtt/test/unit/iot_tests_mqtt_receive.c

```diff
@@ -569,11 +647,12 @@ TEST_SETUP( MQTT_Unit_Receive )
                                                                     1 ) );
 
     /* Clear functions called flags. */
-    _deserializeOverrideCalled = false;
     _getPacketTypeCalled = false;
     _getRemainingLengthCalled = false;
     _networkCloseCalled = false;
     _disconnectCallbackCalled = false;
+
+    IOT_FUNCTION_EXIT_NO_CLEANUP();
```

Comment:
would this macro try to return a status? If so is TEST_SETUP returning anything?

Reply:
Addressed